tutorials/image/cifar10_estimator/cifar10_main.py

```diff
@@ -336,32 +343,24 @@ def _experiment_fn(run_config, hparams):
 
     train_steps = hparams.train_steps
     eval_steps = num_eval_examples // hparams.eval_batch_size
-    examples_sec_hook = cifar10_utils.ExamplesPerSecondHook(
-        hparams.train_batch_size, every_n_steps=10)
-
-    tensors_to_log = {'learning_rate': 'learning_rate', 'loss': 'loss'}
 
-    logging_hook = tf.train.LoggingTensorHook(
-        tensors=tensors_to_log, every_n_iter=100)
-
-    hooks = [logging_hook, examples_sec_hook]
+    if run_config.num_worker_replicas:
+      num_workers = run_config.num_worker_replicas + 1
```

Comment:
Well.   But this +1 is due to a bug.  I could just fix the bug today or tomorrow?

Alternatively, I'll have to remove this +1 when the bug is fixed.

--
Bug: third_party/tensorflow/contrib/learn/python/learn/estimators/run_config.py line=393 needs count in 'master' in addition to 'worker'.

Reply:
Yeah. Either way. Up to you.

Reply:
Eli, this +1 shouldn't be needed as of 168702095.